react-components/stories/index.stories.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
-
-import { storiesOf } from '@storybook/react';
-import { action } from '@storybook/addon-actions';
-
-import ToolDetailsTest from './apps/details/ToolDetails.stories'
-import CategoryTreeTest from './apps/details/CategoryTree.stories'
-import CopyTextAreaTest from './util/CopyTextArea.stories'
+import React from "react";
+import {storiesOf} from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import ToolDetailsTest from "./apps/details/ToolDetails.stories";
+import CategoryTreeTest from "./apps/details/CategoryTree.stories";
+import CopyTextAreaTest from "./util/CopyTextArea.stories";
+import BasicDetailsTest from "./data/details/BasicDetails.stories";
+import InfoTypeSelectionListTest from "./data/details/InfoTypeSelectionList.stories";
+import TagTest from "./data/details/Tag.stories";
+import TagPanelTest from "./data/details/TagPanel.stories";
+import AppStatsTest from "./apps/admin/AppStats.stories";
+import DEHyperLinkTest from "./util/hyperlink/DEHyperLink.stories";
 
 storiesOf('apps/details', module).add('CategoryTree', () => <CategoryTreeTest logger={action('hierarchy')} />);
 storiesOf('apps/details', module).add('ToolDetails', () => <ToolDetailsTest/>);
 storiesOf('util', module).add('CopyTextArea', () => <CopyTextAreaTest/>);
+storiesOf('util', module).add('DEHyperLink', () => <DEHyperLinkTest/>);
+
+
+storiesOf('apps/admin/AppStats', module).add('with test stats', () => <AppStatsTest/>);
+
+storiesOf('data/BasicDetails', module).add('with test diskresource details', () => <BasicDetailsTest logger={action('details')}/>);
```

Comment:
I think it would help keep things organized if these were alphabetical.
```jsx
storiesOf('apps/admin', module)...
storiesOf('apps/details', module)...
storiesOf('data/...', module)...
storiesOf('util', module)...
```
Especially as our list of components starts to grow, it will also make it easier to find individual stories.

Reply:
👍